libcontainer/container_linux.go

```diff
@@ -1498,6 +1498,8 @@ func (c *linuxContainer) criuSwrk(process *Process, req *criurpc.CriuReq, opts *
 	if err := cmd.Start(); err != nil {
 		return err
 	}
+	// we close criuServer so that even if CRIU crashes or unexpectedly exits, runsc will not hang.
```

Comment:
runsc -> runc?

Reply:
Let's leave runsc, it is more secure :-D

Reply:
I'm not sure why you are talking about runsc. Is runsc vendoring this code?

Reply:
@avagin 
I searched the codebase.
runsc doesn't appear elsewhere.

Should we use runc instead ?

Reply:
Sorry Guys. It was my failing joke. We definitely need to fix this typo. It is appeared here due to my typo in the comment in #2284. In our days, I write runsc much more often than runc.